lib/TPP/Conversion/ConvertLinalgToFunc/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+add_mlir_conversion_library(MLIRLinalgToFunc
```

Comment:
I would not use `MLIR` in the library name, as we may accidentally conflict with the upstream naming. Use `TPPLinalgToFunc` etc.